tools/slang-test/slang-test-main.cpp

```diff
@@ -3272,6 +3270,19 @@ SlangResult innerMain(int argc, char** argv)
     
     Options& options = context.options;
 
+    // Set up the prelude
+    {
+        TestToolUtil::PreludeInfo info;
+        info.exePath = argv[0];
+
+        if (options.nvAPIPath.getLength())
+        {
+            info.nvAPIPath = options.nvAPIPath.getBuffer();
```

Comment:
Having the path to NVAPI sit in the `PreludeInfo` for all targets feels a little odd, but I can see how it doesn't make sense to prematurely trying to generalize this to handling arbitrary platform-specific SDKs when we only need to deal with one right now.